Public/Src/Engine/Scheduler/PipHistoricPerfData.cs

```diff
@@ -176,23 +177,30 @@ public PipHistoricPerfData MakeFresh()
         /// </summary>
         public PipHistoricPerfData Merge(PipHistoricPerfData other)
         {
-            return new PipHistoricPerfData(
-                DefaultTimeToLive,
-                GetMergeResult(DurationInMs, other.DurationInMs),
-                GetMergeResult(PeakMemoryInKB, other.PeakMemoryInKB),
-                (ushort)GetMergeResult(ProcessorsInPercents, other.ProcessorsInPercents),
-                GetMergeResult(DiskIOInKB, other.DiskIOInKB));
+            var durationResult = GetMergeResult(DurationInMs, other.DurationInMs);
+            var peakMemoryResult = GetMergeResult(PeakMemoryInKB, other.PeakMemoryInKB);
+            var processorInPercentResult = GetMergeResult(ProcessorsInPercents, other.ProcessorsInPercents);
+            var diskIOResult = GetMergeResult(DiskIOInKB, other.DiskIOInKB);
+
+            return new PipHistoricPerfData(DefaultTimeToLive, durationResult, peakMemoryResult, (ushort)processorInPercentResult, diskIOResult);

```

Comment:
What's the change here?  Only reformatting?

Reply:
Sort of, before the stack trace could not indicate any line number hence its was not easy to say which counter caused a potential overflow.